util-tests/src/test/resources/test-cases/wiredInherited.success

```diff
@@ -1,6 +1,7 @@
 #include commonSimpleClasses
 
 trait Test1 {
+    protected val aA = wire[A]
```

Comment:
does including this protected member change the test somehow? If so, I'd put it in a separate test

Reply:
this is to show that protected members are not taking part in dependency resolution when inherited (at least currently)

Reply:
ok, I think it would be best then to move it to a separate test, sth link `wireOnlyPublic`

Reply:
this is still testing 'inheritance'
non-public members are only ignored when coming from super classes

Reply:
ok, right, so `wireInheritedOnlyPublic` :)

it is a special-case that we are testing here

Reply:
we'd be testing the same thing twice

Reply:
well one test is the basic case - inherited members are used for wiring
the second is more specialised - inherited protected members are *not* used for wiring

Reply:
added a test in 'tests' project